docs/spec/interfaces.md

```diff
@@ -0,0 +1,93 @@
+# Interface Contracts
+
+<!-- This image is sourced from https://drive.google.com/open?id=1Mgn0Oz85M9W1Pwfnw3u0OhZmIIxSxcM3lZUEZyPCoB0 -->
+![Interface Contracts Overview](images/interface-contracts-overview.svg)
+
+---
+
+## Subscribable
+
+A **Subscribable** resource will emit events that a _Subscription_ can direct
+to a _Targetable_ or _Sinkable_ resource.
+
+### Control Plane
+
+A **Subscribable** resource may be referenced in the _from_ section of a
+_Subscription_. The **Subscribable** resource MUST expose a
+_subscribable.channelable_ field (an ObjectReference) in its _status_ section.
+This _channelable_ field MUST be _Channelable_, and _channel_ MAY refer back to
+the **Subscribable** resource.
+
+### Data Plane
+
+A **Subscribable** resource is an event producer or forwarder, events that it
```

Comment:
s/is an event forwarder or producer/forwards or produces events/

Preference for active voice.

Reply:
took all of these.